### ISSUE TITLE
Installer error when upgrading from 2.3.7 to 2.4.0

### ISSUE THREAD

**Issue by linquize** [git-for-windows/git#134]
Installer error when upgrading from 2.3.7 to 2.4.0

```
Error
C:\Program Files\Git\etc\package-versions.txt

An error occurred while trying to rename a file in the destination directory:
MoveFile failed; code 183.
Cannot create a file when that file already exists.

Click Retry to try again, Ignore to skip this file (not recommended), or Abort to cancel installation.

Abort   Retry   Ignore   
```

Platform: Win 7 x64, git-for-windows: 64-bit

**Comment by dscho** [comment]
This file was written into `/etc/packages.txt/packages.txt` by mistake, and I fixed that bug in the newest installer. The overwrite error will most likely fixed automatically once I addressed #107.

**Comment by dscho** [comment]
As #107 is fixed, this issue will be fixed with the next release, too.
